router/tests/query_params.rs

```diff
@@ -0,0 +1,52 @@
+use cfg_if::cfg_if;
+
+cfg_if! {
+    if #[cfg(feature = "ssr")] {
```

Comment:
Not sure if there is any way to test wasm code with JS invocations rich now :/

Reply:
It's possible using `wasm-pack test` or `cargo test` using the `wasm32-unknown-unknown` target, but not as part of our CI.